### ISSUE TITLE
"." characters in file path cause errors

### ISSUE THREAD

**Issue by thehirou** [KosmosisDire/obsidian-webpage-export#94]
Just updated to the latest version and tried to export, this error keeps popping up. It did pop up before with a different note but after closing, opening and trying again it got interrupted by a different one. I tried Self contained documents, multi file website, a different folder etc...

##### [Error] Could not export file: Notes Things X would say.md
```
Error: vaultRelativeDestination must be a folder
    at new Downloadable (plugin:webpage-html-export:10600:13)
    at ExportFile.getSelfDownloadable (plugin:webpage-html-export:12162:12)
    at HTMLGenerator.generateWebpage (plugin:webpage-html-export:11723:33)
    at async HTMLExportPlugin.exportFile (plugin:webpage-html-export:12306:7)
    at async HTMLExportPlugin.exportFolder (plugin:webpage-html-export:12346:28)
    at async HTMLDivElement.eval (plugin:webpage-html-export:12249:24)
```

**Comment by KosmosisDire** [comment]
This looks to be related to #90. Does your file's path have any "." characters in it? This is a bug which I need to fix. If it does not have any "."s in it then it would be really helpful if you could give me the path to that file. Thanks!

**Comment by thehirou** [comment]
Well, this is the path, as far as I'm aware it doesn't have any "." in it
C:\Users\Marco\Documents\GitHub\test

**Comment by bernardb2010** [comment]
> Well, this is the path, as far as I'm aware it doesn't have any "." in it
> C:\Users\Marco\Documents\GitHub\test

This is the path to your vault, correct?

I was having this issue because folders within my vault were labeled, for example, "3. Meetings"

Any note directly inside that folder would cause the issue. However, if there was a subfolder ("3. Meetings/general"), the general subfolder and notes within it were able to export.

**Comment by thehirou** [comment]
> > Well, this is the path, as far as I'm aware it doesn't have any "." in it
> > C:\Users\Marco\Documents\GitHub\test
> 
> This is the path to your vault, correct?
> 
> I was having this issue because folders within my vault were labeled, for example, "3. Meetings"
> 
> Any note directly inside that folder would cause the issue. However, if there was a subfolder ("3. Meetings/general"), the general subfolder and notes within it were able to export.

Nope, that's the path to where I was exporting, the path to my vault is this though
C:\Users\Marco\Documents\Drive Sync\Apfel Project

Maybe it has something to do that these files are being synced by google drive?

**Comment by KosmosisDire** [comment]
Ah okay, I need to know the full path to the file in your vault. Not just the vault path, and not the path you are exporting to

**Comment by thehirou** [comment]
Wooooops, sorry for the trouble, there was a folder named "Misc." where the troubling note was. I thought you meant the path I was exporting the html to.

**Comment by KosmosisDire** [comment]
No problem, I wasn't very clear on that!

**Comment by KosmosisDire** [comment]
This is fixed in update 1.6.0, let me know if you have any more problems! And thanks for the report :)
